.travis.yml

```diff
@@ -56,14 +56,13 @@ script:
   - export TEST_EXTRA=""
   - if [[ "$COVERAGE" == "true" ]]; then
       export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-    fi
-  - if [[ "$STATIC" == "false" ]]; then
+    elif [[ "$STATIC" == "true" ]]; then
```

Comment:
Oh, I thought that was the point. I thought that STATIC and COVERAGE would never both be true. Also, in the install section above, it's an elif.